hpx/lcos/async.hpp

```diff
@@ -158,43 +193,23 @@ namespace hpx { namespace detail
         }
     };
 
-    ///////////////////////////////////////////////////////////////////////////
+    // BOOST_SCOPED_ENUM(launch)
     template <typename Policy>
     struct async_dispatch<Policy,
```

Comment:
The local case should be implemented in `hpx/async.hpp` and the remote in `hpx/lcos/async.hpp`, right? This seems to be the case already, the only thing I am wondering is if the implementation of `async_dispatch` shouldn't go into `hpx/async.hpp` instead, as it is the case for all others. In the, it shouldn't really matter. `hpx/lcos/async.hpp` is included by `hpx/async.hpp` anyways. I was thinking about the corner case where you only want to include the local async variants, which seems to be impossible at the moment anyways.